float32 nav_roll
float32 nav_pitch
float32 nav_bearing

float32 target_bearing
float32 wp_dist
float32 xtrack_error
float32 turn_distance		# the optimal distance to a waypoint to switch to the next

float32 landing_horizontal_slope_displacement
float32 landing_slope_angle_rad
float32 landing_flare_length
bool abort_landing
